LambdaEngine/Source/Game/ECS/Systems/Networking/InterpolationSystem.cpp

```diff
@@ -33,24 +33,24 @@ namespace LambdaEngine
 	void InterpolationSystem::Tick(Timestamp deltaTime)
 	{
 		ECSCore* pECS = ECSCore::GetInstance();
-		auto* pInterpolationComponents = pECS->GetComponentArray<InterpolationComponent>();
-		auto* pPositionComponents = pECS->GetComponentArray<PositionComponent>();
+		auto* pInterpolationComponents	= pECS->GetComponentArray<InterpolationComponent>();
+		auto* pPositionComponents		= pECS->GetComponentArray<PositionComponent>();
 
 		Timestamp currentTime	= EngineLoop::GetTimeSinceStart();
 		float64 percentage;
 
 		for (auto& entity : m_InterpolationEntities)
 		{
 			InterpolationComponent& interpolationComponent	= pInterpolationComponents->GetData(entity);
-			PositionComponent& pPositionComponent			= pPositionComponents->GetData(entity);
+			PositionComponent& positionComponent			= pPositionComponents->GetData(entity);
 
 			deltaTime = currentTime - interpolationComponent.StartTimestamp;
 			percentage = deltaTime.AsSeconds() / interpolationComponent.Duration.AsSeconds();
 			percentage = percentage > 1.0f ? 1.0f : percentage < 0.0f ? 0.0f : percentage;
 
-			Interpolate(interpolationComponent.StartPosition, interpolationComponent.EndPosition, pPositionComponent.Position, (float32)percentage);
+			Interpolate(interpolationComponent.StartPosition, interpolationComponent.EndPosition, positionComponent.Position, (float32)percentage);
 			
-			pPositionComponent.Dirty = true;
+			positionComponent.Dirty = true;
```

Comment:
Dirty flags are automatically set to true if const is omitted when calling ->GetData(entity)
```suggestion
```